### PR TITLE
Major update of "How to host"

### PR DESCRIPTION
**Pull request by robpalme**
There's a bunch of information we talk about with candidate hosts. So this is an attempt to capture more of it in the primary location for reuse.

### CODE REVIEW

File: host.md [hunk 1]
```diff
@@ -1,36 +1,138 @@
...
+- Recording/streaming of the event is possible so long as that is communicated in advance and individuals who do not wish to be captured can opt-out.
+
+## Timeline of Events
+
+This is the general timeline of events leading up to the meeting.
+
+1. Reach out to the chairs for early discussions.
+1. Book the meeting room and confirm you have budget secured.
+1. The chairs will announce the meeting, normally as part of the annual schedule.
+1. 3-4 months before the meeting... the Chairs will conduct an Interest Survey to determine go/no-go.
```
Comment (nicolo-ribaudo) [inline host.md:135]: Why `...` in this sentence and in the next ones?

Comment (robpalme) [reply host.md:135]: It was meant to just be a recurring visual separation of date from activity. Happy to delete if you think it's not adding value.

File: host.md [hunk 2]
```diff
@@ -1,36 +1,138 @@
...
 
-## During the Meeting
+Full details are below.
+The big ticket items you need budget for, and access to, are:
 
-1. Give a short presentation on the facilities - where to find the bathroom, how to use the WiFi, etc.
+- A room that can seat ~30 people for three days that has a large display to show slides and the video call.
+- High quality internet, speakers, and microphones for the video call
+- Catering for breakfast and lunch over the three days
+- Budget for one social dinner for all attendees
```
Comment (rkirsling) [inline host.md:17]: nit: "budget for" is already before the colon. Maybe something like:
```suggestion
- One social dinner outing for all attendees
```

**Review by ryzokuken — APPROVED** [review]

**Comment by robpalme** [comment]
All comments have been resolved so far.  I'd like to merge this next week if there are no further comments.

**Pull request merged**